src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/FileStationProxy.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Download.Clients.DownloadStation.Responses;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Download.Clients.DownloadStation.Proxies
+{
+    public class FileStationProxy : DiskStationProxyBase, IFileStationProxy
+    {
+        public FileStationProxy(IHttpClient httpClient, Logger logger)
+            : base(httpClient, logger)
+        {
+        }
+
+        public IEnumerable<int> GetApiVersion(DownloadStationSettings settings)
+        {
+            return base.GetApiVersion(settings, SynologyApi.FileStationList);
+        }
+
+        public string GetPhysicalPath(string sharedFolder, DownloadStationSettings settings)
+        {
+            var arguments = new Dictionary<string, object>
+            {
+                { "api", "SYNO.FileStation.List" },
+                { "version", "2" },
+                { "method", "getinfo" },
+                { "path", $"[\"{ sharedFolder }\"]" },
+                { "additional", $"[\"real_path\"]" }
+            };
+
+            try
+            {
+                var response = ProcessRequest<FileStationListResponse>(SynologyApi.FileStationList, arguments, settings);
+
+                if (response.Success == true)
+                    return response.Data.Files.First().Additional["real_path"].ToString();
```

Comment:
Add `{ ... }`

(I know, when I first joined the Sonarr team I had to get used to the coding style as well :wink:)